src/test/java/org/openrewrite/java/testing/jmockit/JMockitDelegateToMockitoTest.java

```diff
@@ -0,0 +1,364 @@
+/*
+ * Copyright 2023 the original author or authors.
+ * <p>
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ * <p>
+ * https://www.apache.org/licenses/LICENSE-2.0
+ * <p>
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.openrewrite.java.testing.jmockit;
+
+import org.junit.jupiter.api.Disabled;
+import org.junit.jupiter.api.Test;
+import org.openrewrite.DocumentExample;
+import org.openrewrite.test.RecipeSpec;
+import org.openrewrite.test.RewriteTest;
+
+import static org.openrewrite.java.Assertions.java;
+import static org.openrewrite.java.testing.jmockit.JMockitTestUtils.setDefaultParserSettings;
+
+/**
+ * At the moment, JMockit Delegates are not migrated to mockito. What I'm seeing is that they are being trashed
+ * with the template being printed out. These tests were written to try to replicate this issue, however I was unable to.
+ * They may help anyone who wants to add Delegate migration.
+ */
+@Disabled
+class JMockitDelegateToMockitoTest implements RewriteTest {
+
+    @Override
+    public void defaults(RecipeSpec spec) {
+        setDefaultParserSettings(spec);
+    }
+
+    @DocumentExample
+    @Test
+    void whenNoArgsVoidMethod() {
```

Comment:
I'm wondering what the right issue to tag these tests with is; would that be 
- https://github.com/openrewrite/rewrite-testing-frameworks/issues/325
```suggestion
    @Issue("https://github.com/openrewrite/rewrite-testing-frameworks/issues/325")
    void whenNoArgsVoidMethod() {
```

Reply:
I am thinking this one: https://github.com/openrewrite/rewrite-testing-frameworks/issues/522 - have also mentioned it in PR description (just added it today). But ... 325 works well too - I'll let you decide.

Oh thanks - I wasn't sure how to tag an issue in this repo, in the past I've done it in a different way, thanks for showing it to me.